dev-resources/vectors/README.md

```diff
@@ -1 +1,15 @@
-These vectors were taken from Kalium's test suite.
+The vectors in the following folders were taken from Kalium's test suite:
```

Comment:
Same comment as on the other PR; please merge master in, delete this README, add a README into the specific directory for the test vector you're adding.

Reply:
Done.